lib/go-rfc/mimetype_test.go

```diff
@@ -0,0 +1,359 @@
+package tc
```

Comment:
package rfc - this is what is causing the build failure i believe

Reply:
whoops